tsconfig.json

```diff
@@ -1,12 +1,4 @@
 {
   "extends": "./build/tsconfig.common.json",
-  "include": [
-    "docs/src",
-    "build",
-    "gulp",
-    "packages/react/src",
-    "packages/react/test",
-    "perf/src",
-    "types"
-  ]
+  "include": ["docs/src", "build", "gulp", "packages/*/src", "packages/*/test", "perf/src", "types"]
```

Comment:
here is actually a point of my concerns: currently what do we have is that each repo will have the same `tsconfig` - while, at the same time, it would be more beneficial to split these configs for different packages (and, probably, leave the absolutely bare necessary `tsconfig.common` at the root level):

### now
```
tsconfig.common.json
tsconfig.json
  packages
     react
     themes
     ...
```

### better

```
tsconfig.common.json
  packages
     react
        tsconfig.json
     themes
        tsconfig.json
     ...
```

Here are the reasons why it would be beneficial:
- each project will have an autonomy on introducing specific configuration aspects that are project-specific (like 'include' prop)
- it will make packages independent of each other in the aspect of TS transform. Just, as a matter of contrived example, but still - what if there will be a package that is not written in TS? In that case we won't like it even being considered by TS compiler
- we will improve perf of individual builds - as only the paths that are relevant to the project will be included

Reply:
Yep, I fully support you're there. I'm going to do this in next PRs